lib/VisualGenerator.js

```diff
@@ -89,7 +89,104 @@ function copyVisualTemplate(targetPath, templateName) {
     fs.copySync(path.join(VISUAL_TEMPLATES_PATH, '_global'), targetPath);
     fs.copySync(path.join(VISUAL_TEMPLATES_PATH, templateName), targetPath);
 }
+/**
```

Comment:
There is a call to validTemplate function before it.  

 if (!validTemplate(buildOptions.template)) {
                return reject(new Error('Invalid template'));
            }